pkg/service/repair/controller.go

```diff
@@ -92,19 +92,14 @@ func (c *defaultController) allowance(hosts []string) allowance {
 		Replicas: hosts,
 		Ranges:   math.MaxInt32,
 	}
-	if i < 1 {
-		a.ShardsPercent = i
-	}
 
 	for _, h := range hosts {
 		var v int
 		switch {
 		case i == maxIntensity:
 			v = c.limits[h].Max
-		case i < 1:
-			v = c.limits[h].Default
 		default:
-			v = int(i) * c.limits[h].Default
+			v = int(i)
```

Comment:
So here, if i == 0, we set it to max_repair_ranges_in_parallel. Otherwise, we set it to the number specified by the user.

Reply:
correct